openpype/hosts/fusion/addon.py

```diff
@@ -60,8 +60,18 @@ def get_launch_hook_paths(self, app):
             return []
         return [os.path.join(FUSION_HOST_DIR, "hooks")]
 
-    def add_implementation_envs(self, env, _app):
+    def add_implementation_envs(self, env, app):
         # Set default values if are not already set via settings
+        app_version = get_fusion_version(app.name)
+        if int(app_version) < 18:
+            python_path = env["PYTHONPATH"]
```

Comment:
Let's also add a comment explaining what we're trying to do here - plus maybe add a `# todo` to clean it up later whenever we can or have better packaging tooling available for OpenPype/AYON's launching architecture.